src/main/java/uk/ac/cam/cl/dtg/segue/comm/EmailManager.java

```diff
@@ -80,681 +80,80 @@
-        contentProperties.put("sig", SIGNATURE);
-
-        EmailCommunicationMessage e = constructMultiPartEmail(userDTO.getId(), userDTO.getEmail(),
-                emailContent, contentProperties, EmailType.SYSTEM);
-        this.filterByPreferencesAndAddToQueue(userDTO, e);
-    }
-
-    /**
-     * @param givenName
-     *            - users given name
-     * @param familyName
-     *            - users family name
-     * @param emailAddress
-     *            - the email address of the user
-     * @param subject
-     *            - the subject of the email
-     * @param message
-     *            - message from user
+     * Function that enables contact us messages to be sent to a random email address (not to a known user).
+     *
      * @param recipientEmailAddress
-     *            - email address this email is being sent to
-     * @param replyToAddress
-     *            - the email address we want to be replied to
-     * @param replyToName
-     *            - the name to use for Reply-To
-     * @param userId
-     *            - the user ID (if known) of the user
+     *            - Email Address to send the contact us message to.
+     * @param emailValues
+     *            - users family name
      * @throws ContentManagerException
      *             - if some content is not found
      * @throws SegueDatabaseException
      *             - if the database cannot be accessed
      */
-    public void sendContactUsFormEmail(final String givenName, final String familyName,
-            final String emailAddress, final String subject, final String message,
-            final String recipientEmailAddress, final String replyToAddress, final String replyToName,
-                                       @Nullable final String userId)
+    public void sendContactUsFormEmail(final String recipientEmailAddress, final Map<String, Object> emailValues)
             throws ContentManagerException, SegueDatabaseException {
 
         EmailTemplateDTO emailContent = getEmailTemplateDTO("email-contact-us-form");
-        emailContent.setReplyToEmailAddress(replyToAddress);
-        emailContent.setReplyToName(replyToName);
-        emailContent.setSubject("(Contact Form) " + subject);
-
-        Properties contentProperties = new Properties();
-        contentProperties.put("contactGivenName", givenName == null ? "" : givenName);
-        contentProperties.put("contactFamilyName", familyName == null ? "" : familyName);
-        contentProperties.put("contactUserId", userId == null ? "" : userId);
-        contentProperties.put("contactEmail", emailAddress == null ? "" : emailAddress);
-        contentProperties.put("contactSubject", subject == null ? "" : subject);
-        contentProperties.put("contactMessage", message == null ? "" : message);
-        contentProperties.put("sig", SIGNATURE);
+        emailContent.setReplyToEmailAddress(recipientEmailAddress);
```

Comment:
[By which I mean why on earth is the reply to address the recipient's address. Unless we really do mean `no-reply@isaacphysics.org` :P ]